### ISSUE TITLE
Combine and limit delete-notification pushes

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/notifications#936]
Currently all mobile clients always receive all delete-pushes. iOS ignores silent background notifications if there are too many, so we plan to limit and combine them:
- [ ] Only send talk to talk apps and all others to the files app
- [ ] Combine the multiple deletes into 1 push, so instead of:
 ```php
			[
				'nid' => 13,
				'delete' => true,
			]
```
 ```php
			[
				'nid' => 42,
				'delete' => true,
			]
```
 ```php
			[
				'nid' => 23,
				'delete' => true,
			]
```

the Talk iOS app receives:
 ```php
			[
				'nids' => [13, 42, 23],
				'delete-multiple' => true,
			]
```

**Comment by blizzz** [comment]
moving to 25

**Comment by SystemKeeper** [comment]
Fixed by https://github.com/nextcloud/notifications/pull/1429 and included in 26.0.0 ?
